### DISCUSSION TITLE
Integrate Mathematics extension

### DISCUSSION THREAD

**Discussion by albertobenavides** [awcodes/filament-tiptap-editor#185] [Q&A]
Hello, I want to integrate [Mathematics](https://tiptap.dev/api/extensions/mathematics) extension to this filamentphp TipTap Editor. I have my pro key, etc. 

Can you help me?

**Comment by awcodes** [comment]
Try the section in the README. Should be a straight forward integration.

[https://github.com/awcodes/filament-tiptap-editor#custom-extensions](https://github.com/awcodes/filament-tiptap-editor#custom-extensions)
